packages/geocoder/src/apis/here/index.ts

```diff
@@ -0,0 +1,169 @@
+import { stringify } from "querystring"
+import { normalize } from "@conveyal/lonlat"
+
+// Prettier does not support typescript annotation
+// eslint-disable-next-line prettier/prettier
+import type { LonLatOutput } from "@conveyal/lonlat"
+import type { AutocompleteQuery, ReverseQuery, SearchQuery } from "../../geocoders/abstract-geocoder"
+
+const GEOCODE_URL = "https://geocode.search.hereapi.com/v1/geocode"
+const AUTOCOMPLETE_URL = "https://autosuggest.search.hereapi.com/v1/autosuggest"
+const REVERSE_URL = "https://revgeocode.search.hereapi.com/v1/revgeocode"
+
+type HereQuery = {
+  at?: string,
+  in?: string,
+  limit?: number | string,
+  q?: string,
+  qq?: string,
+  lang?: string,
+  politicalView?: string,
+  show?: string,
+  apiKey: string
+}
+
+// These types are standardized for the other geocoders in this library.
+// Perhaps we could extract them out somewhere and reuse them in the other libraries? 
+type HereFetchArgs = {
+  options: RequestInit // Built-in Typing
+  query: HereQuery
+  url: string
+}
```

Comment:
I was confused about the same thing! I I still am not sure why disable-next-line would break linting for the whole file. Is there a newer version of prettier that can handle `import type`?

Reply:
Probably, but someone would have to write a PR so that only modified files are linted, to avoid failing CI jobs unnecessarily.